helm-charts/sep-service/example_values.yaml

```diff
@@ -126,7 +126,7 @@ app_config:
   sep1.enabled: true
   # `sep1.toml.value` is being set by the helm chart from `toml_config`.
   sep10.enabled: true
-  sep10.home_domain: <your_anchor_domain.com>
+  sep10.web_auth_domain: <your_anchor_domain.com>
```

Comment:
This should be something like "the domain hosting this server"

Reply:
Changed.